framework/src/wifi_manager/wifi_manager.c

```diff
@@ -17,30 +17,214 @@
  ****************************************************************************/
 
 #include <tinyara/config.h>
-#include <debug.h>
-#include <fcntl.h>
-#include <mqueue.h>
-#include <stdio.h>
-#include <arpa/inet.h>
+#include <pthread.h>
 #include <wifi_manager/wifi_manager.h>
 #include <protocols/dhcpc.h>
 #include <protocols/dhcpd.h>
-#include <slsi_wifi/slsi_wifi_api.h>
-#include <semaphore.h>
+#ifdef CONFIG_ENABLE_IOTIVITY
+#include <mqueue.h>
+#include <fcntl.h>
+#endif
 #include "wifi_utils.h"
-#include "wifi_mutex.h"
-#include "wifi_net.h"
 
-static void *g_dhcp_handle = NULL;
-static dhcp_sta_joined g_new_sta_join = NULL;
+#define NUM_HANDLE_FUNC 10 // the number of states
+enum _wifimng_state {
```

Comment:
The abbreviate manager is not mng but mngr or mgr.
Please use mngr or mgr.

Reply:
I will rename